githubmarkdownui/ci.py

```diff
@@ -54,12 +90,54 @@ def ci_task_list(self, status: Optional[CIStatus] = None) -> str:
 
         :param status: Only tasks with the given status will be displayed. If not given, all tasks will be displayed
         """
-        pass
+        # Want to display the results like this, and have the additional info in each line aligned with each other:
+        # <emoji>  <task name> (<task duration>)   <additional info>
+        # Need to find the longest emoji + task name + task duration and use that value to left justify each line.
+        longest_string_length = 0
+        for task in self.tasks:
+            if status and task.metadata.status != status:
+                continue
+
+            string_length = task.get_left_justified_task_string_length()
+
+            if string_length > longest_string_length:
+                longest_string_length = string_length
+
+        return code_block('\n'.join([task.build_task_string(longest_string_length) for task in self.tasks
+                                    if not status or status and task.metadata.status == status]))
+
+    def ci_job_tree(self, status: Optional[CIStatus] = None) -> str:
```

Comment:
Can we make it clear in the function name that it only outputs child ci jobs? Maybe `child_ci_job_tree`?

Reply:
good idea, done